templates/customers/verify-code.liquid

```diff
@@ -1,16 +1,19 @@
 {% layout settings.customer_layout %}
-
 <div class="grid">
 
   <div class="grid-item large--one-third push--large--one-third text-center">
+    {% form 'verify_code' %}
```

Comment:
I have an error "Unknown form type". Is this backend stuff?

Reply:
Need to use vc-storefront-core branch [feature/197-2FA-login-workflow](https://github.com/VirtoCommerce/vc-storefront-core/tree/feature/197-2FA-login-workflow) as i wrote in Teams. You need to get it and run storefront from it.